sql/core/src/main/scala/org/apache/spark/sql/execution/command/tables.scala

```diff
@@ -935,10 +935,8 @@ case class ShowTablePropertiesCommand(table: TableIdentifier, propertyKey: Optio
  */
 case class ShowColumnsCommand(
     databaseName: Option[String],
-    tableName: TableIdentifier) extends RunnableCommand {
-  override val output: Seq[Attribute] = {
-    AttributeReference("col_name", StringType, nullable = false)() :: Nil
-  }
+    tableName: TableIdentifier,
+    override val output: Seq[Attribute]) extends RunnableCommand {
```

Comment:
do we need `override val`?

Reply:
> do we need `override val`?

Same change as last PR https://github.com/apache/spark/pull/31341/files
![image](https://user-images.githubusercontent.com/46485123/106468154-87439980-64d8-11eb-89cb-c04c8316d323.png)